spec/marten/cli/manage/command/collect_assets_spec.cr

```diff
@@ -108,5 +110,41 @@ describe Marten::CLI::Manage::Command::CollectAssets do
 
       File.exists?("spec/assets/css/test.css").should be_true
     end
+
+    it "copies the assets as expected, adds a fingerprint and creates the correct mapping inside a manifest.json" do
```

Comment:
Can we also add a similar unit testing the `--fingerprint` option but without setting the `--manifest-path` option (so that we check that the manifest is created in the expected default folder)?